### PR TITLE
Decoupling of OPA start up with policy loading

### PR DESCRIPTION
**Pull request by ritwikc**
In this PR, two issues have been addressed. 

[1] OPA start up has been decoupled with policy loading.
[2] The access type function is passed in the form of a string in wkcpolicy.rego

### CODE REVIEW

**Comment by ritwikc** [comment]
Comments from PR #298 will now be addressed in this PR.

**Comment by roee88** [comment]
Hi @ritwikc, is this still a draft or ready for review? I see things like `irltest3` and `wkc` that you use for testing.

**Comment by ritwikc** [comment]
I think for testing purposes those have been used. I think the rest is ready for review (not a draft version).

File: third_party/opa/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+kubectl create configmap wkcpolicy.rego  --from-file=data_policies -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego  --local openpolicyagent.org/data=opa | kubectl apply -f -
+
+
+
+
+
+
+kubectl create configmap wkcpolicy.rego --from-file=wkcpolicy.rego -n irltest3 -o yaml --dry-run=client | kubectl label -f- --dry-run=client -o yaml --local openpolicyagent.org/policy=rego | kubectl apply -f -
```
Comment (froesef) [inline third_party/opa/README.md:21]: When storing all the policies etc in configmaps please keep in mind that there is a limit of 1MiB per configmap. So for larger policy definitions there may be problems.

Comment (roee88) [reply third_party/opa/README.md:21]: It's configmap per rego, as used in opa kube-mgmt.

Comment (roee88) [reply third_party/opa/README.md:21]: Generally see https://ibm.github.io/the-mesh-for-data/docs/usage/opa/update_policies/ for what we have today (before this PR)

**Comment by ritwikc** [comment]
Removed irltest3 and wkc. Please review it.

**Review by shlomitk1 — APPROVED** [review]

**Comment by ritwikc** [comment]
To be adrressed later. Currently to go with PR #338

**Pull request closed without merging**